Aquality.WinAppDriver.Template/Aquality.WinAppDriver.Template.SpecFlow/StepDefinitions/CalculatorWindowSteps.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Aquality.WinAppDriver.Template.Forms;
+using NUnit.Framework;
+using TechTalk.SpecFlow;
+
+namespace Aquality.WinAppDriver.Template.SpecFlow.StepDefinitions
+{
+    [Binding]
+    public sealed class CalculatorWindowSteps
+    {
+        private readonly CalculatorForm calculatorForm;
+
+        public CalculatorWindowSteps()
+        {
+            calculatorForm = new CalculatorForm();
```

Comment:
probably better to resolve it from BoDi, receiving as constructor parameter. Same could work  for other steps and windows